__tests__/affirmation.test.tsx

```diff
@@ -0,0 +1,16 @@
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import Page from '../app/affirmation/page'
+
+jest.mock('next/navigation', () => jest.requireActual('next-router-mock'))
+jest.mock('../lib/zoomapi', () => jest.requireActual('../lib/fakezoomapi'));
+
+describe('Page', () => {
+  it('renders a heading', () => {
+    render(<Page />)
+
+    const heading = screen.getByRole('heading', { level: 1 })
+
+    expect(heading).toBeInTheDocument()
+  })
+})
```

Comment:
I think it would be nice to start adding tests that cover the functionality of the code we are adding (in addition to these simple tests that rendering doesn't fail).

Do you think you could add a test for a small piece of interactive functionality?  I'm thinking something like "when you click apply on an affirmation, the affirmation at the top of the page changes".